packages/viewer/typedoc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "entryPoints": [
+    "./src/index.ts",
+    "./src/commands/index.ts",
+    "./src/interactions/index.ts",
+    "./src/scenes/index.ts",
+    "./src/types/index.ts"
```

Comment:
including these as `entryPoints` tells typedoc to generate documentation for the classes/interfaces, then link to them in the viewer docs. see `Scene` here https://vertexvis.github.io/vertex-web-sdk/packages/viewer/interfaces/viewer.components.vertexviewer.html#scene

another approach would be to define these directories as modules and export them in `src/index.ts` as well, if it's valuable to keep a single entry point.

@danschultz